rita.rst

```diff
@@ -23,10 +21,6 @@ Warning
 
 Please keep in mind we do not officially support RITA, so installation is at your own risk.
 
-Additionally, RITA currently only supports use of Zeek logs in ``TSV`` format. If you are running the latest version of Security Onion, you will need to switch from ``JSON`` to ``TSV`` format by following the steps here:
-
-`<Zeek#tsv>`__
```

Comment:
With v3.2.0 we've added the ability to parse JSON files and have tested them from Security Onion.